lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_betabuild_prechecks.rb

```diff
@@ -91,7 +91,7 @@ def self.authors
       end
 
       def self.is_supported?(platform)
-        platform == :ios
+        [:ios, :mac.include?(platform)
```

Comment:
```suggestion
        [:ios, :mac].include?(platform)
```

Reply:
You actually got the same typo everywhere, so I'm going to let you do the find-and-replace rather than commenting on each 😉

Reply:
@AliSoftware Yes, thank you! 😅 🤦‍♂️ Hey, it's a draft still 😜

Reply:
Ah whoops 😅  Received a notification when you added us as reviewer, and didn't think twice nor noticed it was still Draft before going straight to review, sorry 😄